CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ All notable changes to [`@bpmn-io/properties-panel`](https://github.com/bpmn-io/
 
 ___Note:__ Yet to be released changes appear here._
 
+* `FEAT`: improve FEEL popup lifecycle events:
```

Comment:
We could consider to make this less verbose, and simply link to the (comprehensive) PR description. We want to link to a PR anyway.